src/components/BookItem.jsx

```diff
@@ -1,19 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
```

Comment:
- [ ] Another issue shows up on the terminal because of `prop-types`. I would recommend adding 'prop-types' to the dependencies in `packege.json` file to fix this issue:

<img width="671" alt="image" src="https://user-images.githubusercontent.com/77190203/160720324-6e9cce51-1bef-4990-a77f-4c5c7ab3b2f6.png">